configs/testdata/warning-files/ignore_changes.tf

```diff
@@ -1,11 +0,0 @@
-resource "null_resource" "one" {
-  lifecycle {
-    ignore_changes = ["triggers"] # WARNING: Quoted references are deprecated
```

Comment:
I assume this is being dropped because it's going to become an error in a follow-up PR. Is that correct?

Reply:
Yep! There's also other files that contain this warning (so even if it were missed, it would be ok!). There will be a `testdata/error-files` for quoted references in the PR that upgrades that to an error.